File: euler/javascript/euler2.js
var prev1;
var prev2 = 1;
var curr = 1;
var sum = 0;

while (curr <= 4000000) {

  if (curr % 2 === 0) {
    sum = sum + curr;
  }

  prev1 = prev2;
  prev2 = curr;
  curr = prev1 + prev2;
}

console.log(sum);

File: euler/python/euler2.py
x = 1
y = 1
sum = 0

while x < 4000000:
    new = x + y
    x = y
    y = new
    if x % 2 == 0:
        sum = sum + x

answer = 4613732

print(sum)
print(answer == sum)
